### DISCUSSION TITLE
Argument to skip Video already transcripted

### DISCUSSION THREAD

**Discussion by oevesque** [Purfview/whisper-standalone-win#47] [Ideas]
Hi,
When using wildcard as argument (ex: *.mp4) it would be great if there is some argument so skip file who have already output srt files.

**Comment by Purfview** [comment]
It's simple, don't transcribe files which are already transcribed. :)

**Comment by oevesque** [comment]
Of course :-)
But I want to launch Whisper-faster periodically on Radar/sonar download folder without writing code to identify which file don't have subtitle.

**Reply by Purfview** [reply]
Implemented in r141.3 with `--skip` argument.

**Reply by oevesque** [reply]
thanks for your work.
